pki.spec

```diff
@@ -1160,6 +1163,11 @@ fi
 %pom_disable_module console base
 %endif
 
+# configure target Java version
+%pom_xpath_set \
+    "pom:properties/pom:maven.compiler.release" \
+    "$JAVA_VERSION"
+
```

Comment:
I do not think the compiled bytecode release should be related to the default JVM available in the build environment. I think we should set a value based on the language and support all the version from that.